packer.json

```diff
@@ -12,11 +12,11 @@
       "source_ami_filter": {
         "filters": {
           "virtualization-type": "hvm",
-          "name": "Windows_Server-2012-R2_RTM-English-64Bit-Base-*",
+          "name": "base-win-2012-wherescape-*",
           "root-device-type": "ebs"
         },
         "owners": [
-          "801119661308"
+          "093401982388"
```

Comment:
you probably want prod+notprod in this list

Reply:
Prod+notprod will not use this directly but the encrypted copy right?

Reply:
you need to share the ami with prod+notprod so that it can be copied there

Reply:
it is shared at the ami_users section. Owners refers to the owner of the AMI which is CI.

      "ami_users": [
        "483846886818",
        "337779336338"
      ]

Reply:
Yes. Sorry. You’re quite right